package.json

```diff
@@ -1,22 +1,31 @@
 {
   "name": "bee-queue",
-  "version": "0.3.0",
+  "version": "1.0.0",
   "description": "A simple, fast, robust job/task queue, backed by Redis.",
   "main": "index.js",
   "dependencies": {
-    "redis": "1.0.0"
+    "promise-callbacks": "^3.0.0",
```

Comment:
I'd much prefer if we pin these. I'm guessing it's just oversight, but I'm happy to discuss if you'd explicitly rather not.